cards/src/UserCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import getUsers from './getUsers';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: 1000,
+    maxWidth: 545,
+  },
+});
+
+const UserCard = () => {
+  const [user, setUser] = React.useState({
+    avatar:'loading...',
+    email:'loading...',
+    first_name: 'loading...',
+    last_name:'loading...'
+  });
+let number = Math.floor(Math.random() * 12) + 1  
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  React.useEffect(() => {
```

Comment:
I tend to prefer to import my hooks from React in the import statement because I get tired to typing `React.` Some people prefer to just import the default and call this directly, so totally up to you!